RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -12,7 +12,7 @@ @implementation RCTMapboxGL
 
 @end
 
-@interface MGLAnnotation ()
+@interface addGLAnnotation ()
```

Comment:
This should be called something else, like `RCTMGLAnntation`. Classes in frameworks/components should always start with an uppercase prefix, either the project initials or your personal initials.